README.md

```diff
@@ -3,6 +3,15 @@
 
 This is an app that is used to inform local authorities of what t-levels are assigned to-do and what future careers in can align with.The tool is made to integrate with labour market information sources so that users can see job market trends which then allows the user to make informed decisions about education and employment. Local authorities benefit from this integration as it allows them to tailor educational programs to meet the needs of the local job market. On the other hand employers can use the app to identity and recruit talent directly from the pool of students who have completed relevant t-level qualifications.
 
+## Deployment
+For a quick link to the deployment see the top of the README. For more info on the deployment look [here](https://github.com/enBloc-org/pathways/blob/dev/Deployment-Info.md)
+
+## User Guide
+When you arrive at the landing page you can either directly use the search bar to start looking for T-level placements that match your interest. From the landing page or any other page you can also click the about button and see the information about the purpose of the Pathways tool. If you do use the search bar you can search for as many terms as you want just separate them with commas. Once you search your matches if there are any T-level courses relevant to your search. You can click on any of these matches to see more information about the T-level course. If there are lots of results you can filter them by career category. If you want to revisit a search click the save button and your search and filters will be saved for you to view later on this computer. If you want to share search results to another device or with another person copy the URL and visiting that URL on another browser will load up the same results.
```

Comment:
This first sentence needs a revision, as it has a either statement but then doesn't present another option. Needs a rewrite. Also the flow of the last few sentences could be a bit smoother, instead of each one starting with "If" as this sounds a bit repetitive. I think the last two sentences could be blended together with an or for instance.

Reply:
I agree I gave the end part a reword can you take a look at it @nichgalzin